generative_ai/prompts/test_prompt_template.py

```diff
@@ -11,10 +11,60 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from vertexai.preview import prompts
 
+
+import prompt_create_save_generate
+import prompt_delete
+import prompt_list_prompts
+import prompt_list_version
+import prompt_load_or_retrieve
+import prompt_restore_version
 import prompt_template
 
 
 def test_prompt_template() -> None:
     text = prompt_template.prompt_template_example()
     assert len(text) > 2
+
+
+def test_prompt_create_save_generate() -> None:
+    response = prompt_create_save_generate.prompt_local_prompt_create()
+    assert response
+
+
+def test_prompt_list_prompts() -> None:
+    list_prompts = prompt_list_prompts.list_prompt()
+    assert list_prompts
+
+
+def test_prompt_load_or_retrieve() -> str:
```

Comment:
nit: Your method here doesn't have a return, so the return type hint should still be None.